.github/workflows/cla.yml

```diff
@@ -0,0 +1,29 @@
+name: "CLA Assistant"
+
+on:
+  issue_comment:
+    types:
+      - "created"
+  pull_request_target:
+    types:
+      - "opened"
+      - "closed"
+      - "synchronize"
+
+jobs:
+  cla-assistant:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "CLA Assistant"
+        if: "(github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'"
+        uses: "cla-assistant/github-action@v2.1.3-beta"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          PERSONAL_ACCESS_TOKEN: "${{ secrets.CLA_ASSISTANT_ACCESS_TOKEN }}"
+        with:
+          remote-organization-name: "localstack"
+          remote-repository-name: "localstack"
+          path-to-signatures: "v1/signatures.json"
```

Comment:
```suggestion
          path-to-signatures: "etc/cla-signatures/signatures.json"
```

Reply:
Better put the signatures under some top-level folder like `etc` or `doc`, to keep the repo root cleanly structured.